From 15c47a6102e07f846cca2a565525aca15f277b92 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 16 Mar 2020 11:35:44 +0800
Subject: [PATCH] mm: slub: Fix compilation warnings when !SLUB_SYSFS and
 SLUB_DEBUG=y

mm/slub.c:4459:13: warning: unused function 'validate_slab_cache' [-Wunused-function]
mm/slub.c:4618:12: warning: unused function 'list_locations' [-Wunused-function]

Fixes: d2014f8b6800 ("mm: slub: Add SLUB_SYSFS")
Change-Id: Ic933970940a11d1d7dce418f3e49797688998ede
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 init/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/Kconfig b/init/Kconfig
index 8ecbe4d0dd5f..41f50f0ec334 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -1755,6 +1755,7 @@ config SLUB_DEBUG
 	default y
 	bool "Enable SLUB debugging support" if EXPERT
 	depends on SLUB && SYSFS
+	select SLUB_SYSFS
 	help
 	  SLUB has extensive debug support features. Disabling these can
 	  result in significant savings in code size. This also disables
-- 
2.35.3

